client/packages/programs/src/JsonForms/components/Search/Search.tsx

```diff
@@ -81,6 +82,8 @@ const UIComponent = (props: ControlProps) => {
     props.uischema.options
   );
 
+  if (zErrors) return <Typography color="error">{zErrors}</Typography>;
```

Comment:
Errors weren't being displayed by the Child components, as they were never rendered due to missing "options".